Gemfile

```diff
@@ -46,7 +46,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails', '~> 3.5'
   gem 'faker'
-  gem 'pry', '~> 0.12.2'
+  gem 'pry-byebug'
```

Comment:
ooo what's this

Reply:
it fixes a bug I was encountering where I couldn't use the 'next' and 'continue' pry commands